src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -128,6 +128,9 @@ public function handleError($type, $msg, $file, $line, $context = [])
         if ($deprecation->isMuted()) {
             return null;
         }
+        if ($this->getConfiguration()->isBaselineDeprecation($deprecation->getMessage())) {
```

Comment:
I suggest passing the Deprecation object here rather than just the message, which would open the door to more powerful baseline (we could allow deprecations from some locations, but not allowing new places to trigger the same, as done in PHPStan, for instance)

Reply:
Good idea.